### PR TITLE
Wakahn branch3 - adding workbook for Azure resource locations

### PR DESCRIPTION
**Pull request by WKahnZA**
image

Need to confirm how to only show resources with a valid location (lat/lon)

### CODE REVIEW

**Comment by WKahnZA** [comment]
Also please confirm settings.json is correct

File: Workbooks/Azure Resources/Azure Resource Locations/Azure Resource Locations.workbook [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+          {
+            "id": "bfe50469-7469-4c4d-a33e-0edbb43546c4",
+            "version": "KqlParameterItem/1.0",
+            "name": "Subscription",
+            "type": 6,
+            "isRequired": true,
+            "multiSelect": true,
+            "quote": "'",
+            "delimiter": ",",
+            "query": "where type =~ 'microsoft.compute/virtualmachines'\r\n| summarize Count = count() by subscriptionId\r\n\t| order by Count desc\r\n\t| extend Rank = row_number()\r\n\t| project value = subscriptionId, label = subscriptionId, selected = Rank == 1",
```
Comment (gardnerjr) [inline Workbooks/Azure Resources/Azure Resource Locations/Azure Resource Locations.workbook]: this query is only showing subs that have virtual machines.  intentional?

Comment (WKahnZA) [reply Workbooks/Azure Resources/Azure Resource Locations/Azure Resource Locations.workbook]: Not intentional will update today

Comment (WKahnZA) [reply Workbooks/Azure Resources/Azure Resource Locations/Azure Resource Locations.workbook]: Updated to include all subscriptions

**Comment by WKahnZA** [comment]
Also improved colour scheme
image

image

File: Workbooks/Azure Resources/Azure Resource Locations/settings.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+    "$schema": "https://github.com/Microsoft/Application-Insights-Workbooks/blob/master/schema/settings.json",
+    "name":"Azure Resource Locations",
+    "author": "Microsoft",
+    "galleries": [
+        { "type": "workbook", "resourceType": "Azure Monitor", "categoryKey": "All", "order": 100  }
```
Comment (acearun) [inline Workbooks/Azure Resources/Azure Resource Locations/settings.json]: You already have a categoryResources.json at the lower level - you don't need a categoryKey. The category string (Azure Resources) will be picked from categoryResources.json.

Comment (WKahnZA) [reply Workbooks/Azure Resources/Azure Resource Locations/settings.json]: categorykey from settings.json

**Comment by gardnerjr** [comment]
looks good, couple of questions:

1) why not allow all subs?  why a limit of 100?  azure resource graph allows up to 1000 subs, so setting the limit to 1000 and allowing all would probably let most people use "all" ?

2) similar question with resource types, why not allow multi select there and also support all?

**Comment by WKahnZA** [comment]
Updated with recommendations above

**Comment by WKahnZA** [comment]
reduced subscription limit to 100 until it can be raised

**Pull request merged**